static/app/views/alerts/rules/metric/details/relatedIssues.tsx

```diff
@@ -115,7 +115,6 @@ function RelatedIssues({
 
       <TableWrapper>
         <GroupList
-          orgSlug={organization.slug}
```

Comment:
Since I needed the full `organization`, I removed all references to the `orgSlug` prop